setup.py

```diff
@@ -146,7 +150,10 @@ def get_long_description():
                   'run = avocado.plugins.run:Run',
                   'sysinfo = avocado.plugins.sysinfo:SysInfo',
                   'plugins = avocado.plugins.plugins:Plugins',
-                  ]
+                  ],
+              'avocado.plugins.job.prepost': [
+                  'jobscripts = avocado.plugins.jobscripts:JobScripts',
+                  ],
```

Comment:
In general I like this idea, but I think it should be connected with moving the `sysinfo` hooks to this infrastructure and removing it. For non-daemons the compatibility could be kept by symlink (for few releases) and for daemons a special version of `pre/post` script would have to be implemented.

I mean it probably can coexists for a while, but please add a trello card reference to get rid of it and mark it as `to-be-removed` so people start looking at this instead.